### PR TITLE
fixes https://github.com/fluentribbon/Fluent.Ribbon/issues/117

### PR DESCRIPTION
**Pull request by janmolnar**

### CODE REVIEW

File: Fluent/Controls/Backstage.cs [hunk 1]
```diff
@@ -378,6 +398,33 @@ private void CreateAndAttachBackstageAdorner()
 
             var layer = AdornerLayer.GetAdornerLayer(this);
             layer.Add(this.adorner);
+
+            layer.CommandBindings.Add(new CommandBinding(RibbonCommands.OpenBackstage,
```
Comment (batzen) [inline Fluent/Controls/Backstage.cs]: This does not seem to be related to your fix. Why are you adding a new CommandBinding to the layer? The Backstage itself already has such a CommanBinding.

File: Fluent/Controls/Backstage.cs [hunk 2]
```diff
@@ -378,6 +398,33 @@ private void CreateAndAttachBackstageAdorner()
...
+                }));
+        }
+
+        /// <summary>
+        /// Gets the first visual child of type TChildItem by walking down the visual tree.
+        /// </summary>
+        /// <typeparam name="TChildItem">The type of visual child to find.</typeparam>
+        /// <param name="obj">The parent element whose visual tree shall be walked down.</param>
+        /// <returns>The first element of type TChildItem found in the visual tree is returned. If none is found, null is returned.</returns>
+        private TChildItem FindVisualChild<TChildItem>(DependencyObject obj) where TChildItem : DependencyObject
```
Comment (batzen) [inline Fluent/Controls/Backstage.cs]: Please move this method to the UIHelper class.

File: Fluent/Controls/Backstage.cs (hunk 1 above)
Comment (janmolnar) [inline Fluent/Controls/Backstage.cs]: However the backstages CommandBindings do not work when the window content is an arbitrary .NET Object. Thus I removed the commandbindings from the backstage and added it to the layer

Von meinem iPhone gesendet

Am 25.05.2015 um 14:17 schrieb Bastian Schmidt <notifications@github.com<mailto:notifications@github.com>>:


In Fluent/Controls/Backstage.cs<https://github.com/fluentribbon/Fluent.Ribbon/pull/118#discussion_r30976996>:

> @@ -378,6 +398,33 @@ private void CreateAndAttachBackstageAdorner()
>
>              var layer = AdornerLayer.GetAdornerLayer(this);
>              layer.Add(this.adorner);
> +
> +            layer.CommandBindings.Add(new CommandBinding(RibbonCommands.OpenBackstage,


This does not seem to be related to your fix. Why are you adding a new CommandBinding to the layer? The Backstage itself already has such a CommanBinding.

-
